libs/libqmi/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libqmi
 PKG_VERSION:=1.26.6
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
If you're already updating the package with this change, why bump PKG_RELEASE?

Reply:
A very welcome change, @aleksander0m !  Thanks for your hard work.  Correct, if the PKG_VERSION increases, then PKG_RELEASE resets to 1.

Reply:
@neheb @nickberry17 the thing is that this PR has 3 commits, one doing `1.26.2-1 -> 1.26.4-1`; another one doing `1.26.4-1 -> 1.26.6-1`; and the last one adding the new config option, bumping PKG_RELEASE: `1.26.6-1 -> 1.26.6-2`. I didn't want to have the config change in the same commit as the version bump, as it was a completely orthogonal thing. Would it be better if I just send one single commit with everything? I personally prefer the 3 separate commits, but whatever you prefer.

Reply:
I mean up to you. Typically, a PR either bumps the PKG_RELEASE or resets it to 1 as anything intermediate is not present.

Reply:
If it's up to me, I'd prefer to leave it as 3 separate commits, each bumping only the version field that needs to be bumped. In my mind a PR is a collection of commits, but each commit needs to be self-contained by itself. Makes it easier e.g. to cherry-pick to the 19.07 branch if we want to cherry-pick the version bump only and not the new config change.